wrapper/pre_copy.py

```diff
@@ -202,7 +204,13 @@ def auth_cb(cred, _):
 
         cred_info = [[libvirt.VIR_CRED_AUTHNAME, libvirt.VIR_CRED_PASSPHRASE],
                      auth_cb, None]
-        conn = libvirt.openAuth(self._uri, cred_info)
+
+        if re.compile('^ssh://').match(self._uri):
```

Comment:
All examples I've seen were using `esx://user@host/`, is the `ssh://` uri used as well somewhere?

Reply:
`ssh://` is virt-v2v construct